bytecomp/bytegen.ml

```diff
@@ -576,7 +576,7 @@ let rec comp_expr env exp sz cont =
         in
         comp_init env sz decl_size
       end
-  | Lprim((Pidentity | Popaque), [arg]) ->
+  | Lprim((Pidentity | Popaque | Pbytes_to_string | Pbytes_of_string), [arg]) ->
```

Comment:
the goal is propagating the information while **not impacting the semantics** of code generator for this revision(it already touched 25 files and needs bootstrapping), once we change the code generation, it might introduce some subtle bugs if it is not complete.
As I said, it is a good step to propagate it into bytecode/native backend once you have confidence that it is complete, but I won't do it in this revision